specs/_features/eip7805/p2p-interface.md

```diff
@@ -54,7 +54,7 @@ The following validations MUST pass before forwarding the `inclusion_list` on th
 - _[IGNORE]_ The slot `message.slot` is equal to the current slot, or it is equal to the previous slot and the current time is less than `ATTESTATION_DEADLINE` seconds into the slot.
 - _[IGNORE]_ The `inclusion_list_committee` for slot `message.slot` on the current branch corresponds to `message.inclusion_list_committee_root`, as determined by `hash_tree_root(inclusion_list_committee) == message.inclusion_list_committee_root`.
 - _[REJECT]_ The validator index `message.validator_index` is within the `inclusion_list_committee` corresponding to `message.inclusion_list_committee_root`.
-- _[REJECT]_ The transactions `message.transactions` length is within upperbound `MAX_TRANSACTIONS_PER_INCLUSION_LIST`.
+- _[REJECT]_ The transactions `message.transactions` length is within upperbound `MAX_TRANSACTIONS_PER_PAYLOAD`.
```

Comment:
is `MAX_TRANSACTIONS_PER_PAYLOAD` just a placeholder for now? since this is 1048576 on mainnet it seems meaningless to have this check, however it might be good to come up with a good limit on the ssz container anyways?

otherwise I think we should just limit the inclusion list by size in bytes via `MAX_BYTES_PER_INCLUSION_LIST` and reconsider other limits later

Reply:
IMO we could remove this check for now and keep the one based on `MAX_BYTES_PER_INCLUSION_LIST`

Reply:
I removed the transactions count checks since the size the size is set at the container level. The p2p bytes size check would be sufficient imo